src/sentry/models/group.py

```diff
@@ -624,9 +623,6 @@ def save(self, *args, **kwargs):
             self.message = truncatechars(self.message.splitlines()[0], 255)
         if self.times_seen is None:
             self.times_seen = 1
-        self.score = type(self).calculate_score(
-            times_seen=self.times_seen, last_seen=self.last_seen
-        )
```

Comment:
Let's also remove the `calculate_score` method?

Reply:
removed it and `get_score` as well. I was also able to remove `ScoreClause` from `src/sentry/event_manager.py`which I believe is unused as well.